Packs/Cylance_Protect/Integrations/Cylance_Protect_v2/Cylance_Protect_v2.yml

```
@@ -960,14 +985,63 @@ script:
     - contextPath: Cylance.Device.Deleted
       description: A boolean to check if the device was deleted.
       type: boolean
-    description: Delete one or more devices from an organization.
-    execution: true
+  - arguments:
+    - default: false
+      description: The MAC Address of the device
+      isArray: false
+      name: mac
+      required: true
+      secret: false
+    deprecated: false
+    description: Allows a caller to request a specific device resource belonging to
+      a Tenant by MAC Address
+    execution: false
+    name: cylance-protect-get-devicebymac
```

let's change that to 
```suggestion
    name: cylance-protect-get-device-by-mac
```